chokuretsu/guide/play-game.html

```diff
@@ -0,0 +1,248 @@
+                            using its' SD card or connector cable.
+                        </li>
+                        <li>Insert the flash cartridge into your Nintendo DS console.</li>
+                        <li>Turn on your system and run the <code>dsbf_dump.nds</code> application</li>
+                        <li>Wait for the firmware dump to complete. Then, power off your Nintendo DS, remove the cart
+                            and insert the flash cart's SD card into your computer (or connect it with the correct
+                            cable).
+                        </li>
+                        <li>Your flash cartridge's SD card/storage will now contain <code>BIOSNDS7.ROM</code>, <code>BIOSNDS9.ROM</code>
+                            and a <code>.BIN</code> file starting with the letters "FW". These are your Nintendo DS'
+                            ARM7 and ARM9 BIOS files and its firmware respectively.
+                        </li>
+                    </ol>
+                </li>
+                <li>Copy your ARM7, ARM9 and firmware files into the same directory as where you downloaded melonDS
+                    to.
+                </li>
+                <li>Open melonDS, then on the top bar menu click "Config" <i class="fa-solid fa-arrow-right"></i> "Emu
+                    settings"
+                </li>
+                <li>On the "General" tab, ensure Console Mode is set to "DS". Click through to the tab named "DS-mode"
+                </li>
+                <li>On each of the required BIOS and firmware files, select the ARM7 <code>.ROM</code>, ARM9
+                    <code>.ROM</code> and firmware <code>.BIN</code> files you ripped from your system.
+                </li>
+            </ol>
+            <b>Playing the game:</b>
+            <p>With melonDS setup, all you need to do in order to play the game is select "File" <i
+                    class="fa-solid fa-arrow-right"></i> "Open ROM" in the top menu bar and choose the ROM file you
+                patched earlier.</p>
+            <p>melonDS has a number of advanced features, such as configuring controllers and tweaking display
+                settings. You can check out the <a href="https://melonds.kuribo64.net/" target="_blank"
+                                                   class="exclude-device-href">melonDS Website</a> for more information.
+            </p>
+
+            <h4 id="other-pc-emulators">Other PC emulators</h4>
+            <ul>
+                <li><a href="https://desmume.org/" target="_blank" class="exclude-device-href">DeSumME</a>&#65293;Not
+                    recommended due to having issues with sound playback on cutscenes. If you can overlook that, it's
+                    still a popular emulator with good community support and options.
+                </li>
+            </ul>
+
+            <!-- Wii U VC emulator -->
+            <h3 id="wii-u-vc">Emulate on Wii U by injecting the Virtual Console</h3>
```

Comment:
Aight looked into this now too.

* https://gbatemp.net/threads/release-uwuvci-injectiine.486781/
* https://flumpster.github.io/instructions/nds/nds.html
* https://gbatemp.net/threads/release-wup-installer-gx2-rpx-channel.456199/ or https://github.com/dimok789/loadiine

Not sure how to get the common key and title keys without just looking them up online tho, so we'll have to look into that.